charts/cloudzero-agent/README.md

```diff
@@ -154,12 +151,6 @@ kube-state-metrics:
   image:
     registry: my-custom-registry.io
     repository: my-custom-kube-state-metrics/kube-state-metrics
-
-prometheus-node-exporter:
-  enabled: true
-  image:
-    registry: my-custom-registry.io
-    repository: my-custom-prometheus/node-exporter
 ```
 
 ### Custom Scrape Configs
```

Comment:
Do we need this section at all? I believe this is now taken care of in the validator - no?

Reply:
good catch, removed